nox.py

```diff
@@ -61,7 +61,6 @@ def unit(session, py):
     session.run(
         'py.test',
         '--quiet',
-        '--cov=opencensus',
```

Comment:
You don't need to specify a project? I've been running with `opencensus` for the core library and `opencensus.ext` for the contrib packages.

Reply:
I was hoping that by removing this line, `[run] source` in `.coveragerc` would take effect, but I realized it was overwritten by `pytest-cov`:
https://github.com/pytest-dev/pytest-cov/blob/41eea2e3e28e6f6849104d6526bb633e638f8a0f/src/pytest_cov/plugin.py#L56